onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4741,7 +4741,16 @@ def aten_topk(
 ) -> tuple[TensorType, TensorType]:
```

Comment:
need: 

```
@torch_op("aten::topk")
```

Reply:
self: TReal, … -> tuple[TReal, INT64]

Reply:
It does both compilation and registration. however this registry is only for code gen and not public facing.